### ISSUE TITLE
[UI] Community Feed

### ISSUE THREAD

**Issue by leecalcote** [meshery/meshery#1751]
### Current Behavior
The "Community" link in Meshery currently opens a new browser window.

### Desired Behavior
The "Community" link should instead open a local page in Meshery that contains a segmented list of noteworthy items in the community.  Items may be an interesting blog post, upcoming event, recent project release, recently news, and so on.

### Implementation
1. Noteworthy items should be dynamically retrieved from a remote endpoint.
1. Noteworthy items should be presented in categories with the following segments: **News** (posts, news), **Events** (talks, workshops), **Releases**
1. Create the `layer5.io/community/feed` endpoint to provided a JSON-formatted list of the ~10 most recent items containing frontmatter variable `community-feed` with value of `true`.

### Acceptance Tests
1. Entries are shown in descending order by date.
1. When Meshery UI deployed in an environment that does not allow or have access to the Internet (to the community news endpoint), then the user is shown a form to subscribe to the community newsletter.
1. The remote endpoint should be set JSON-formatted endpoint should be created and available from layer5.io.

**Comment by leecalcote** [comment]
@MaximeCoignus interested in taking this on?

**Comment by Shulammite-Aso** [comment]
@leecalcote is there still need to work on this?

I see the _community_ link on meshery UI opens up Meshery's slack. This is the link you're referring to right? and not the one on the website [meshery.io](https://meshery.io/)?

**Comment by leecalcote** [comment]
@Shulammite-Aso, yes, there's still effort needed here (no effort has been given as of yet other than describing the request).

We're looking for the "Community" link to be updated or a new "Updates" link to be added to read from an RSS feed (or endpoint that returns a simple json doc or markdown doc) that contains latest community updates (e.g. announcement of a new release, an upcoming webinar, and so on). This new endpoint will need to be created on meshery.io.

**Comment by Shulammite-Aso** [comment]
This makes sense @leecalcote. would like to work on this.  However the endpoint at meshery.io is not ready yet, will i need to create this endpoint myself too?

**Comment by leecalcote** [comment]
@Shulammite-Aso oh, my goodness. I completely missed your comment!

**Comment by Shulammite-Aso** [comment]
Will do this @leecalcote

**Comment by leecalcote** [comment]
@Shulammite-Aso excellent. :) I'm tagging @Nikhil-Ladha and @vineethvanga18 as two individuals to engage in this effort. They will be most supportive collaborators. 👍

**Comment by Shulammite-Aso** [comment]
Great @leecalcote. On slack, I have suggested working on the endpoint too. If it's possible for me to take it up, then i'd appreciate their help a lot. I just like the idea of contributing with Go too.

**Comment by mohakmalviya** [comment]
hello @leecalcote i am interested on working on this issue!

**Comment by leecalcote** [comment]
@mohakmalviya there is an implementation of this in Meshery Cloud that is an ideal reference implementation. Try it out.

// @sudhanshutech

**Comment by leecalcote** [comment]
https://github.com/meshery/meshery/assets/7570704/f81ab72b-3463-4433-a9cb-da8de7bbdb07

**Comment by matrixkavi** [comment]
Closing as implemented. 

<img width="1200" height="640" alt="Image" src="https://github.com/user-attachments/assets/2f440f7d-5288-45a5-9ccc-a01dd3a2a7e0" />
